tests/UnitTestsGowerMetric.py

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
Filenames should be the same as the thing they are testing, prefixed with `test_`, e.g. `test_GowerMetric`